src/backend/gl/src/queue.rs

```diff
@@ -488,16 +491,15 @@ impl CommandQueue {
                     .ClearBufferiv(gl::COLOR, draw_buffer, cv.as_ptr());
             },
             com::Command::ClearBufferDepthStencil(depth, stencil) => unsafe {
-                let (target, depth, stencil) = match (depth, stencil) {
-                    (Some(depth), Some(stencil)) => (gl::DEPTH_STENCIL, depth, stencil),
-                    (Some(depth), None) => (gl::DEPTH, depth, 0),
-                    (None, Some(stencil)) => (gl::STENCIL, 0.0, stencil),
+                let (depth, stencil) = match (depth, stencil) {
+                    (Some(depth), Some(stencil)) => (depth, stencil),
+                    (Some(depth), None) => (depth, 0),
+                    (None, Some(stencil)) => (0.0, stencil),
                     _ => unreachable!(),
                 };
-
                 self.share
                     .context
-                    .ClearBufferfi(target, 0, depth, stencil as _);
+                    .ClearBufferfi(gl::DEPTH_STENCIL, 0, depth, stencil as _);
```

Comment:
why is this hardcoded?

Reply:
Khronos documentation:
> clearBufferfi may be used to clear the depth and stencil buffers. buffer must be DEPTH_STENCIL and drawBuffer must be zero. depth and stencil are the depth and stencil values, respectively.

Reply:
@Thomspoon 
>  **glClearBufferfi and glClearNamedFramebufferfi are used to clear the depth and stencil buffers simultaneously.** buffer must be GL_DEPTH_STENCIL and drawbuffer must be zero. depth and stencil are the values to clear the depth and stencil buffers to, respectively. Clamping and type conversion of depth for fixed-point depth buffers are performed in the same fashion as for glClearDepth. Masking of stencil for stencil buffers is performed in the same fashion as for glClearStencil. These commands are equivalent to clearing the depth and stencil buffers separately, but may be faster when a buffer of internal format GL_DEPTH_STENCIL is being cleared. The same per-fragment and masking operations defined for glClear are applied. 

-- https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glClearBuffer.xhtml

When `depth` or `stencil` is `None`,  `glClearStencil` or `glClearDepth` should be used, respectively.

Reply:
right, it isn't correct but should use the other glClearBuffer variants

Reply:
glClearStencil and glClearDepth aren't correct because we want to clear them on the default framebuffer. I'm using glClearBufferfi to clear both, glClearBufferfv to clear depth and glClearBufferiv to clear stencil as recommended here https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glClearBuffer.xhtml